packages/big-design/src/components/Input/styled.tsx

```diff
@@ -39,48 +33,78 @@ export const StyledInput = styled.input<InputProps>`
           `}
   }
 
-  &:focus {
-    outline: none;
-    box-shadow: 0 0 0 4px ${props => (props.error ? props.theme.colors.danger20 : props.theme.colors.primary20)};
-  }
+  ${({ error, focus, theme }) =>
+    focus &&
+    css`
+      outline: none;
+      box-shadow: 0 0 0 4px ${error ? theme.colors.danger20 : theme.colors.primary20};
+    `};
 
   &[disabled] {
     background-color: ${({ theme }) => theme.colors.secondary20};
   }
+`;
+
+export const StyledInput = styled.input<InputProps>`
+  border: 0;
+  box-sizing: border-box;
+  color: ${({ theme }) => theme.colors.secondary70};
+  flex: 1;
+  height: 26px;
```

Comment:
Any reason for not using `theme.spacing` for the height still?

Reply:
There's no way to calculate `26px` using the current values of `theme.spacing`. Will use `remCalc`.